README.md

```diff
@@ -95,6 +95,7 @@ For reference material, we have provided existing Kubernetes deployment models t
 
 1.  [Seagate Internal Jenkins Job](http://eos-jenkins.mero.colo.seagate.com/job/Cortx-kubernetes/job/setup-kubernetes-cluster/)
 2.  [CORTX on AWS and Kubernetes - Quick Install Guide](doc/cortx-aws-k8s-installation.md)
+    -  [AWS CloudFormation Template](doc/cloudformation.md) for this setup
```

Comment:
Instead of a separate document, would this be better as a set of edits to the original AWS/k8s guide?

Reply:
Just moved it into the original guide